src/com/company/Main.java

```diff
@@ -2,37 +2,25 @@
 
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
-import java.util.Date;
-import java.util.Random;
-import java.util.Scanner;
+import java.util.*;
 
 import static java.time.temporal.TemporalQueries.localDate;
 
 public class Main {
 
     public static void main(String[] args) {
-
+        System.out.println("Type count of chat members");
         Scanner input = new Scanner(System.in);
-        int count = input.nextInt();
-        int activeMembersCount=count;
+        int membersCount = input.nextInt();
+        ArrayList<String> listOfChatMembers = new ArrayList<>(membersCount);
+        for (int i = 1; i <= membersCount; i++) {
+            listOfChatMembers.add("member" + i + " name: ");
+        }
 
         while (true) {
```

Comment:
Try to avoid while(true) in case of you can, for example, here you can check with while(!listOfChatMembers.isEmpty())

Reply:
Why we should avoid while(true)? Are there any restrictions about it? If I'm not mistaken while(true) and while(!listOfChatMembers.isEmpty()) do the same quantity actions. Thank you :)

Reply:
@Lusin-mus  jan it is the same, but I will suggest using while(!listOfChatMembers.isEmpty()) as in case of while(true) when code becomes more complicated it will be hard to maintain, in first case it is clear when that while block will be broken but in case of while true developer should check the whole block to understand the logic

Reply:
I thing in every case developers will check the whole block to understand the logic, but I agree with you that with a concrete condition everything is more comprehensive. So I will change the code. Thank you :)